src/_common/M.js

```diff
@@ -15,11 +15,13 @@ export default class M extends Component {
 		const { m = '', values } = this.props;
 
 		return (
-			<FormattedMessage
-				id={m}
-				defaultMessage={m}
-				values={values}
-			/>
+			<span {...this.props}>
```

Comment:
That was one of my annoyances that they don't pass a className down to their span, but they discussed it on the Issues part of GitHub and was against it. So this is probably better.